packages/pytest-simcore/src/pytest_simcore/traefik_service.py

```diff
@@ -34,7 +36,9 @@ def traefik_endpoints(
 
 @pytest.fixture(scope="function")
 async def traefik_service(
-    loop, traefik_endpoints: tuple[URL, URL, URL], docker_stack: dict
+    event_loop: asyncio.AbstractEventLoop,
```

Comment:
Q: why is the event_loop even needed here?
if the tests are run with asyncio_mode=auto this should not be necessary.

Reply:
Now that you mention ... not sure. I think it is an old addition.